src/components/chat/chat-panel.tsx

```diff
@@ -62,21 +55,18 @@ export const ChatPanel: React.FC<IProps> = ({ user, activeNavTab, focusDocument,
 
   const commentsPathRef = useCurrent(useCommentsCollectionPath(focusDocument || ""));
   const deleteCommentMutation = useDeleteDocument();
+
   const deleteComment = useCallback((commentId: string, commentText: string) => {
-    const storeDocument = focusStoreDocumentRef.current;
     if (focusDocumentRef.current) {
-      if (!focusTileIdRef.current) {
-        if (storeDocument) {
-          Logger.logDocumentEvent(LogEventName.CHAT_PANEL_DELETE_COMMENT_FOR_DOCUMENT, storeDocument, commentText);
-        }
-      } else {
-        const commentTile = storeDocument?.content?.getTile(focusTileIdRef.current);
-        if (commentTile) {
-          Logger.logTileEvent(LogEventName.CHAT_PANEL_DELETE_COMMENT_FOR_TILE, commentTile, undefined, commentText);
-        }
-      }
+      const eventPayload = {
+        focusDocumentId: focusDocumentRef.current,
+        focusTileId: (focusTileIdRef && focusTileIdRef.current) || undefined,
```

Comment:
```typescript
        focusTileId: focusTileIdRef.current,
```
`focusTileIdRef` is always defined and its value will either be a string or `undefined`.